language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/BallerinaTextDocumentService.java

```diff
@@ -108,7 +108,7 @@
     BallerinaTextDocumentService(LSGlobalContext globalContext, WorkspaceManager workspaceManager) {
         this.workspaceManager = workspaceManager;
         this.languageServer = globalContext.get(LSGlobalContextKeys.LANGUAGE_SERVER_KEY);
-        this.diagnosticsHelper = globalContext.get(LSGlobalContextKeys.DIAGNOSTIC_HELPER_KEY);
+        this.diagnosticsHelper = new DiagnosticsHelper();
```

Comment:
Any reason for not using the diagnostic helper instance created at `BallerinaLanguageServer` constructor?

Reply:
I think we didn't do that change coz of the rush we had ... anyway, this is not the whole static PR as well... this is a temp fix to get the diagnostic issue fixed...